jupyter/laceworkjupyter/__init__.py

```diff
@@ -33,12 +33,14 @@ class LaceworkJupyterHelper:
 
     def __init__(
             self, api_key=None, api_secret=None, account=None,
-            subaccount=None, instance=None, base_domain=None):
+            subaccount=None, instance=None, base_domain=None,
+            config_section=None):
```

Comment:
nit. Can we change `config_section` to `profile` to match the terminology from the Lacework CLI? https://github.com/lacework/go-sdk/tree/main/cli#multiple-profiles
```suggestion
            profile=None):
```

Reply:
done